docs/proxy/datacenter_proxy/nodejs_examples.md

```diff
@@ -126,9 +128,15 @@ Use one IP chosen from `SHADER` and `BUYPROXIES94952` proxy groups for the brows
 
     Apify.main(async () => {
         const password = process.env.APIFY_PROXY_PASSWORD;
```

Comment:
Let's remove the `password`. The example only shows what's done automatically.

Reply:
Removed.